splash.html

```diff
@@ -1,22 +1,126 @@
 <!DOCTYPE html>
```

Comment:
Rename this file to game.html if this is the only html file used in the game

Reply:
now game.html